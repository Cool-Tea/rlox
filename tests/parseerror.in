print 1
fun f(a, b {}
var = 5;
@abc
{var x = 1;
