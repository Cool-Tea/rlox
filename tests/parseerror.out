Error at 'EOF': Expect ';' after value.